cleanlab/rank.py

```diff
@@ -494,9 +501,9 @@ def get_self_confidence_for_each_label(
       Lower scores indicate more likely mislabeled examples.
     """
 
-    # np.mean is used so that this works for multi-labels (list of lists)
-    label_quality_scores = np.array([np.mean(pred_probs[i, l]) for i, l in enumerate(labels)])
-    return label_quality_scores
+    # To make this work for multi-label (but it will slow down runtime), replace:
+    # pred_probs[i, l] -> np.mean(pred_probs[i, l])
+    return np.array([pred_probs[i, l] for i, l in enumerate(labels)])
```

Comment:
Can `get_self_confidence_for_each_label` take in a flag `multi_label` to have this work for both?

Reply:
That is not for this PR, but yes good point. 

Can you open a PR pointing users to how they should get label-quality scores for multi-label datasets in these functions (ie. linking to the proper multilabel functions). Note users are not supposed to use these functions here for multilabel datasets.